pkg/api/customization/globalrolebinding/validator.go

```diff
@@ -0,0 +1,23 @@
+package globalrolebinding
+
+import (
+	"net/http"
+
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+)
+
+func Validator(request *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
```

Comment:
Should this also account for both subjects being empty since we don't allow updates to that field

Reply:
it does, it would return an error in that case, I tried to write the error so it would make sense in either scenario